scripts/convert_music_spectrogram_to_diffusers.py

```diff
@@ -28,7 +28,9 @@ def load_token_encoder(weights, model):
         )
 
         attention_weights = ly_weight["attention"]
-        lyr.layer[0].SelfAttention.q.weight = nn.Parameter(torch.FloatTensor(attention_weights["query"]["kernel"].T))
+        lyr.layer[0].SelfAttention.q.weight = nn.Parameter(
+            torch.FloatTensor(attention_weights["query"]["kernel"].T * depth_scaling)
```

Comment:
Let's please not do this - weights should not be scaled just so that we don't show the scaling in the code. This will be very confusing for future weights that want to use this code (e.g. all future weights will also have to be scaled)

Reply:
yes I will revert it! this is not the issue